### ISSUE TITLE
Qwant is not recognized

### ISSUE THREAD

**Issue by Alwaysin** [matomo-org/plugin-GoogleAnalyticsImporter#285]
Initial tentative https://github.com/matomo-org/searchengine-and-social-list/pull/84 but I've been suggested that it is an issue here.

I'm having this message when importing:

```
WARNING [2022-10-05 14:20:59] 20496 Encountered unknown search engine source from Google Analytics: qwant.com
```

**Comment by snake14** [comment]
Hi @Alwaysin . Thank you for taking the time to create this issue. It looks like Stefan has suggested where the change should probably take place. I'm not familiar enough with the plugin to know what else needs to be changed, so I'm going to mark this issue to be prioritised, unless @AltamashShaikh thinks it will be a quick enough change.

**Comment by MatomoForumNotifications** [comment]
This issue has been mentioned on **Matomo forums**. There might be relevant details there:

https://forum.matomo.org/t/warnings-during-import-googleanalyticsimporter/47693/2

**Comment by AltamashShaikh** [comment]
already fixed with #408
